### PR TITLE
GUI development

### PR DESCRIPTION
**Pull request by greyhund**

### CODE REVIEW

File: mainframe.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        }
+
+        @Override
+        protected void paintComponent(Graphics g) {
+            super.paintComponent(g);
+            if (backgroundImage != null) {
+                g.drawImage(backgroundImage, 0, 0, getWidth(), getHeight(), this);
+            }
+        }
+    }
```
Comment (jdlc05) [inline mainframe.java:36]: Add whitespace after this line

File: mainframe.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                showSecondScreen();
+            }
+        });
+        backgroundPanel.add(playButton, BorderLayout.SOUTH);
+
+        mainFrame.setContentPane(backgroundPanel);
+        mainFrame.setVisible(true);
+    }
```
Comment (jdlc05) [inline mainframe.java:89]: Add whitespace after this line

File: mainframe.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    /************************************************************/
+    /*********************SECOND SCREEN**************************/
```
Comment (jdlc05) [inline mainframe.java:91]: Add whitespace after this line

**Review by jdlc05 — CHANGES_REQUESTED** [review]
Please make the quick changes requested, thanks!

**Review by logan82902 — APPROVED** [review]
Great job with the changes, thanks!

**Review by jdlc05 — APPROVED** [review]

**Review by logan82902 — APPROVED** [review]

**Pull request merged**